docs/nb/Median_DMcosmic.ipynb

```diff
@@ -0,0 +1,496 @@
+{
```

Comment:
<p>Are these being used? Also to confirm, this notebook compares the median DM when smearing due to scattering is not considered (FRB) vs when it is (ZDM) right? Or am I confused and this has to do with the host contributions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/FRBs/zdm/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>